scripts/mounts/mysql/init/2_StaticInsertValuesMainBackendless.sql

```diff
@@ -67,6 +67,7 @@ INSERT INTO DeveloperOperation VALUES ('112','ENABLE_PANIC');
 INSERT INTO DeveloperOperation VALUES ('113','MANAGE_AUTH0_SECURITY');
```

Comment:
You forgot to update DBs versions here. Also add new index to script which creates "main_application"